### PR TITLE
Secondary script to check for successful integrations downstream

### PR DESCRIPTION
**Pull request by ghzlatarev**
closes Manta-Network/Manta#56

* Added a script to detect if integrations have been successful -> ``check_integrations_success``
* Clones all sync branches and builds. Expects success.
* Only triggered if the initial integration check (which detects failure) has failed.
* Updated ``check_integrations_failure`` script to update upstream dependencies
* Updated manta dependencies to latest commits

### CODE REVIEW

File: scripts/check_integrations_success.sh [hunk 1]
```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+cd ..
+
+mkdir breaking_change_check
+
+cd breaking_change_check
+
+BRANCH_STRING="some_sync_branch_name"
```
Comment (ghzlatarev) [inline scripts/check_integrations_success.sh:9]: This is the variable you gotta edit manually for the routine to work out.
All the sync branches should use the same name.

**Review by stechu — CHANGES_REQUESTED** [review]
Can you write a doc for the `sync-branch` design?
I don't feel confident to merge the branch before understanding the design.

**Comment by ghzlatarev** [comment]
> Can you write a doc for the `sync-branch` design?
> I don't feel confident to merge the branch before understanding the design.

Please check this doc out:
https://github.com/Manta-Network/Manta/wiki/How-to-update-Manta-dependencies-in-downstream-repositories.

We've already merged this functionality in other repos though.

**Comment by stechu** [comment]
> > Can you write a doc for the `sync-branch` design?
> > I don't feel confident to merge the branch before understanding the design.
> 
> Please check this doc out:
> https://github.com/Manta-Network/Manta/wiki/How-to-update-Manta-dependencies-in-downstream-repositories.
> 
> We've already merged this functionality in other repos though.

I cannot open the link.

**Comment by ghzlatarev** [comment]
> > > Can you write a doc for the `sync-branch` design?
> > > I don't feel confident to merge the branch before understanding the design.
> > 
> > 
> > Please check this doc out:
> > https://github.com/Manta-Network/Manta/wiki/How-to-update-Manta-dependencies-in-downstream-repositories.
> > We've already merged this functionality in other repos though.
> 
> I cannot open the link.

https://github.com/Manta-Network/Manta/wiki/How-to-update-Manta-dependencies-in-downstream-repositories.

> > > Can you write a doc for the `sync-branch` design?
> > > I don't feel confident to merge the branch before understanding the design.
> > 
> > 
> > Please check this doc out:
> > https://github.com/Manta-Network/Manta/wiki/How-to-update-Manta-dependencies-in-downstream-repositories .
> > We've already merged this functionality in other repos though.
> 
> I cannot open the link.

Oops there was a mistake in the link . Should work now. This is the manta wiki.

**Pull request closed without merging**